remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/AsyncPoller.kt

```diff
@@ -3,26 +3,43 @@ package com.rakuten.tech.mobile.remoteconfig
 import androidx.annotation.VisibleForTesting
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.delay
+import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.delay
 import java.util.concurrent.TimeUnit
 
 internal class AsyncPoller @VisibleForTesting constructor(
-    delayInMinutes: Int,
+    delayInSeconds: Int,
     private val scope: CoroutineScope
 ) {
 
-    constructor(delayInMinutes: Int) : this(delayInMinutes, GlobalScope)
+    constructor(delayInSeconds: Int) : this(delayInSeconds, GlobalScope)
 
-    private val delayInMilliseconds = TimeUnit.MINUTES.toMillis(delayInMinutes.toLong())
+    private val delayInMilliseconds = TimeUnit.SECONDS.toMillis(delayInSeconds.toLong())
```

Comment:
If the app tries to set it to zero I'm planning to use the default 3600 secs. What do you think?

Reply:
Yes, you're right.
I'll add handling for this. Thanks

Reply:
Actually, why was this changed to seconds instead of minutes? If we use minutes then we can avoid a situation where they are polling too often.

Reply:
updated the minimum based on the discussion.